apis/nucleus/src/components/listbox/default-properties.js

```diff
@@ -26,15 +51,125 @@ const listdef = {
       ],
     },
   },
+  /**
+   * Show histogram bar.
+   * also requires (qListObjectDef.qFrequencyMode 'V' and frequencyMax) or qListObjectDef.qFrequencyMode 'P'
+   * @type {boolean=}
+   * @default
+   */
   histogram: false,
+  /**
+   * frequencyMax calculation
+   * needed for histogram when not using qListObjectDef.qFrequencyMode: 'P'
+   * use an expression in the form `Max(AGGR(Count([field]), [field]))` (when needed)
+   * or 'fetch' that triggers an extra engine call but needed for library dimension that could change field when using the object
+   * @type { 'fetch' | ValueExpression }
+   */
+  frequencyMax: undefined,
+  /**
+   * Show values as checkboxes instead of as fields.
+   * @type {boolean=}
+   * @default
+   */
   checkboxes: false,
+  /**
+   * enables search.
+   * @type {boolean=}
+   * @default true
+   */
+  searchEnabled: true,
+  /**
+   * show title.
+   * @type {boolean=}
+   * @default true
+   */
+  showTitle: true,
+  /**
+   * start search with wildcard search.
+   * @type {boolean=}
+   * @default false
+   */
+  wildCardSearch: false,
+  /**
+   * layout settings.
+   * @type {object=}
+   */
   layoutOptions: {
+    /**
+     * dense mode.
+     * @type {boolean=}
+     * @default false
+     */
     dense: false,
+    /**
+     * layout mode.
+     * @type { 'singleColumn' | 'grid' }
+     * @default
+     */
     dataLayout: 'singleColumn',
```

Comment:
should probably be optional. but unsure what to change to get that flag in the spec

Reply:
`@default "singleColumn"` could help with that... but I'm not 100% sure

Reply:
No. it has no impact on the generated spec